README.md

```diff
@@ -29,7 +29,7 @@ The [example](./example) folder shows how those components can be assembled to c
 ### Building and Running Locally
 
 In order to use the library components or test the library locally, the project components need to be rolled up and
-dependencies need to be installed initially with the command: `npm install`. From here the application can be built
```

Comment:
I don't think we want to change this for developers building locally, just the pipeline(s)